lighthouse-core/audits/byte-efficiency/uses-long-cache-ttl.js

```diff
@@ -218,7 +238,9 @@ class CacheHeaders extends Audit {
 
         results.push({
           url,
-          cacheControl,
+          // Include cacheControl in results, but cast as any so table types
+          // are happy. cacheControl is not shown in the table so this is OK.
+          cacheControl: /** @type {any} */ (cacheControl),
```

Comment:
thoughts?

Reply:
summoning @brendankenny

Reply:
> cacheControl is not shown in the table so this is OK.

It's not ok, why use a table if you don't want it to be a table, but it's not your fault that you're inheriting this :)

Will you add a `TODO(bckenny): fix DetailsItem` to the comment?